### DISCUSSION TITLE
New Ideas for cQUbe!

### DISCUSSION THREAD

**Discussion by DevashishRaghav** [Sunbird-cQube/community#791] [Ideas]
Dear community members
After completing the in-person bootcamp for cQube in Bangalore, we received exciting new ideas. 
Hence, we are starting this discussion to capture all your ideas (related to enhancements, additions or anything else) that can contribute towards building cQube. 
Expecting a free-flowing discussions on the comments and our team will also keep adding their thoughts.

**Comment by arnob93** [comment]
Suggestion: File Ingestion POST api accepts alternative file-url param and downloads the file from url.
